CrewChiefV4/Audio/Sounds.cs

```diff
@@ -21,18 +21,19 @@ public class SoundCache
         
         private List<String> dynamicLoadedSounds = new List<String>();
         public static Dictionary<String, SoundSet> soundSets = new Dictionary<String, SoundSet>();
-        private Dictionary<String, SingleSound> singleSounds = new Dictionary<String, SingleSound>();
+        private static Dictionary<String, SingleSound> singleSounds = new Dictionary<String, SingleSound>();
         public static List<String> sortedAvailableDriverNames = new List<String>();
         public static List<String> sortedAvailableSounds = new List<String>();
         public static List<String> sortedAvailablePrefixesAndSuffixes = new List<String>();
```

Comment:
I do not see load_sound_files_on_startup renamed or tooltip updated.  I think we need to have this option, for the case where user alt+tabs out of game, and starts CC.  If we preload, we'll affect game perf a bit, so we need to allow people to opt out.  This works unchanged?